### ISSUE TITLE
Django 2.2 breaks migrate_schemas (KeyError: 'plan')

### ISSUE THREAD

**Issue by sviehb** [tomturner/django-tenants#259]
Django 2.2 (django-tenants 2.1.0) seems to break the `migrate_schemas` command.

Traceback:
```
Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.5/dist-packages/django_tenants/management/commands/migrate_schemas.py", line 46, in handle
    executor.run_migrations(tenants=[self.PUBLIC_SCHEMA_NAME])
  File "/usr/local/lib/python3.5/dist-packages/django_tenants/migration_executors/standard.py", line 11, in run_migrations
    run_migrations(self.args, self.options, self.codename, self.PUBLIC_SCHEMA_NAME)
  File "/usr/local/lib/python3.5/dist-packages/django_tenants/migration_executors/base.py", line 36, in run_migrations
    MigrateCommand(stdout=stdout, stderr=stderr).execute(*args, **options)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/base.py", line 83, in wrapped
    res = handle_func(*args, **kwargs)
  File "/usr/local/lib/python3.5/dist-packages/django/core/management/commands/migrate.py", line 147, in handle
    if options['plan']:
KeyError: 'plan'
```

Likely caused by the new "migrations plan" feature:
> Migrations
> The new migrate --plan option prints the list of migration operations that will be performed.
https://docs.djangoproject.com/en/2.2/releases/2.2/

**Comment by tomturner** [comment]
Already fixed in 2.2 which is now released

**Comment by bhuwanpaudelofficial** [comment]
i used this  

DB_ENGINE=django.db.backends.postgresql

 then i got the this error 
  connection.set_schema(schema_name, tenant_type=tenant_type, include_public=False)
    ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'DatabaseWrapper' object has no attribute 'set_schema'
then i update the  the database engine 


DB_ENGINE=django_tenants.postgresql_backend


then this problem is solve by updating the code
